redash/models/base.py

```diff
@@ -15,7 +15,7 @@ def apply_driver_hacks(self, app, info, options):
         options.update(json_serializer=json_dumps)
         if settings.SQLALCHEMY_ENABLE_POOL_PRE_PING:
             options.update(pool_pre_ping=True)
-        super(RedashSQLAlchemy, self).apply_driver_hacks(app, info, options)
+        return super(RedashSQLAlchemy, self).apply_driver_hacks(app, info, options)
```

Comment:
I don't know if it's appropriate to add return here.

Reply:
https://github.com/pallets-eco/flask-sqlalchemy/blob/2.5.1/flask_sqlalchemy/__init__.py#L608 

In other parts of flask_sqlalchemy, there is a dependency on this return value.

Reply:
> https://github.com/pallets-eco/flask-sqlalchemy/blob/2.5.1/flask_sqlalchemy/__init__.py#L608
> 
> In other parts of flask_sqlalchemy, there is a dependency on this return value.


Sorry, i didn't look at the new version of the code, you're right!